### DISCUSSION TITLE
Колонки отображаются неправильного размера

### DISCUSSION THREAD

**Discussion by ToeraHL** [totumonline/totum-ru-issues-and-discussions#283] [Как это сделать?] [answered]
Здравствуйте. Я как администратор меняю размер полей и они меняются, но у других пользователей, которые уже были на этой страницы, ничего не меняется. Сброс кеша не помогает. Как с этим бороться?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Ширины колонок запоминаются в localstorage, поэтому кэш их может не сбрасывать.

Если кликнуть правой кнопкой на Глазе — то таблица отобразится с ширинами колонок по умолчанию.

Тут важно обратить внимание, что вы как администратор должны менять значение **ширины по умолчанию.**

Что бы принудительно сбросить отображение ширин колонок надо задать параметр таблицы [Актуальность настроек полей](https://ru.docs.totum.online/tables#table-settings-fields_actuality)

У всех, у кого дата сохранения параметров видимости полей окажется меньше указанной — настройки видимости и ширин будет сброшены к заданным по умолчанию.
